README.md

```diff
@@ -56,9 +56,9 @@ A BibTeX entry for LaTeX users is
   title = {mip: Comparison of multi-model runs},
   author = {David Klein and Jan Philipp Dietrich and Lavinia Baumstark and Florian Humpenoeder and Miodrag Stevanovic and Stephen Wirth and Pascal Führlich and Oliver Richters and Tonn Rüter},
   doi = {10.5281/zenodo.1158586},
-  date = {2025-03-25},
+  date = {2025-03-27},
   year = {2025},
   url = {https://github.com/pik-piam/mip},
-  note = {Version: 0.154.0},
+  note = {Version: 0.154.0.9001},
```

Comment:
Dont forget to increment to non-dev version. Maybe also turn on the lucode check for incremented version for this package.